### DISCUSSION TITLE
How to do system tray in wails v2.5, icon flashing

### DISCUSSION THREAD

**Discussion by wenjian022** [wailsapp/wails#2866] [Q&A]
I have a monitoring platform, I want to make a wails client on Windows, to notify users in real time of those server problems, but also need to hang the wails client on the system tray, when there is an alarm information, the icon flashes to remind the user, the user clicks the flashing icon, you can wake up the wails client.

**Comment by ykdiy** [comment]
wails v2.5 does not support system tray, v3 will support it
